### ISSUE TITLE
数据集训练后效果没有达到预期，请问是不是我的操作有问题？The effect of the dataset training did not meet expectations. Is there a problem with my operation?

### ISSUE THREAD

**Issue by lihaonanleo** [kaize0409/GCN_AnomalyDetection#8]
您好您的tensorflow代码中在进行数据集训练后，AUC指标在Amazon数据300次迭代大概为0.56；在Enron数据集为0.73（并且在大概60次迭代后，损失函数就没有发生变化了）；在Disney数据集中AUC指标为0.49，基于这些指标模型优势并未展现，不知道是不是我的操作有问题，请您指教，另外还有问题在'BlogCatalog', 'Flickr', 'ACM'数据集找不到gnd，也就是没有图中的标签。

Hello, in your TensorFlow code, after data set training, the AUC index for 300 iterations of Amazon data is approximately 0.56; The Enron dataset was 0.73 (and after approximately 60 iterations, the loss function did not change); The AUC indicator in the Disney dataset is 0.49, and the advantages of these indicator models have not been demonstrated. I wonder if there is a problem with my operation. Please advise me. In addition, there is a problem that gnd cannot be found in the 'BlogCatalog', 'Flickr', and 'ACM' dataset, which means there is no tag in the figure.
![Disney](https://user-images.githubusercontent.com/112804078/229397910-ded76690-0a83-465d-b529-806edde0875e.png)
